RecoLocalTracker/SiStripClusterizer/interface/SiStripClusters2ApproxClustersv1.h

```diff
@@ -0,0 +1,32 @@
+#ifndef RecoLocalTracker_SiStripClusters2ApproxClustersv1_h
+#define RecoLocalTracker_SiStripClusters2ApproxClustersv1_h
+
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "DataFormats/SiStripCluster/interface/SiStripApproximateClusterv1.h"
+#include "DataFormats/SiStripCluster/interface/SiStripCluster.h"
+#include "DataFormats/Common/interface/DetSetVectorNew.h"
+#include "DataFormats/Common/interface/DetSetVector.h"
+
+#include <vector>
+#include <memory>
+
+class SiStripClusters2ApproxClustersv1: public edm::stream::EDProducer<>  {
```

Comment:
I thought it was recommended to have the class declaration and methods implementation in the same file for cmssw `plugins`.

Reply:
fixed